modules/apps/client-extension/client-extension-type-item-selector-web/src/main/java/com/liferay/client/extension/type/item/selector/web/internal/item/selector/CETItemSelectorViewDescriptor.java

```diff
@@ -113,6 +98,41 @@ public boolean isShowBreadcrumb() {
 		return false;
 	}
 
+	private List<CET> _filterCETs(List<CET> cets, String type) {
+		Predicate<CET> filterPredicate = _getFilterPredicate(type);
+
+		if (filterPredicate == null) {
+			return cets;
+		}
+
+		return ListUtil.filter(cets, filterPredicate);
+	}
+
+	private Predicate<CET> _getFilterPredicate(String type) {
+		if (Objects.equals(
+				type, ClientExtensionEntryConstants.TYPE_GLOBAL_JS)) {
+
+			return cet -> {
+				GlobalJSCET globalJSCET = (GlobalJSCET)cet;
+
+				return !StringUtil.equalsIgnoreCase(
+					globalJSCET.getScope(), "company");
+			};
+		}
+		else if (Objects.equals(
+					type, ClientExtensionEntryConstants.TYPE_THEME_CSS)) {
+
+			return cet -> {
+				ThemeCSSCET themeCSSCET = (ThemeCSSCET)cet;
+
+				return !StringUtil.equalsIgnoreCase(
+					themeCSSCET.getScope(), "controlPanel");
```

Comment:
Yeah maybe these scopes can be constants.

Reply:
@ethib137  Agreed. But I won't do this now. I was discussing with Drew regarding creating a way to declare valid values/types for client extensions. Currently, we do not have constants for any of them. I will do this soon and will refactor it.

Is that alright for you?

Reply:
Yep that's fine!